src/inmanta/module.py

```diff
@@ -382,6 +382,10 @@ def __init__(self, path: str, autostd: bool = True, main_file: str = "main.cf",
         It does not include
          * verify if project.yml corresponds to the modules in self.modules
 
+        Instances of this class can be created by in two different ways:
+        1) Via the Project.get() method
+        2) Via the constructor: Always call the Project.set() method after the constructor call.
+
```

Comment:
What worries me here is that `Project.set()` also executes 

``` 
os.chdir(project._path)
plugins.PluginMeta.clear()
loader.unload_inmanta_plugins()
```
while this is not done when `Project.get()` is used. @wouterdb @bartv Should we make sure that `Project.get()` also executes the above-mentioned statements.

It also not entirely clear to me why this class executes `os.chdir(project._path)`. Any idea?

Reply:
no idea

Reply:
The project set is only for testing. 

```
plugins.PluginMeta.clear()
loader.unload_inmanta_plugins()
```
causes the project related singletons to be reset. 

The 

```
os.chdir(project._path)
```

I don't know if we still need that.